app/routes/campaigns.js

```diff
@@ -41,4 +46,51 @@ router.get('/:id', (req, res) => {
     .catch(error => res.send(error));
 });
 
+router.post('/:id/message', (req, res) => {
+  // Check required parameters.
+  const campaignId = req.params.id;
+  const phone = req.body.phone;
+  const type = req.body.type;
+
+  if (!campaignId) {
+    return helpers.sendResponse(res, 422, 'Missing required campaign id.');
+  }
+  if (!phone) {
+    return helpers.sendResponse(res, 422, 'Missing required phone parameter.');
+  }
+  if (!type) {
+    return helpers.sendResponse(res, 422, 'Missing required message type parameter.');
+  }
+
+  // Check that campaign is active.
+  const campaign = app.locals.campaigns[campaignId];
+  if (!campaign || campaign.status !== 'active') {
+    const msg = `Campaign ${campaignId} is not running on CampaignBot`;
+    return helpers.sendResponse(res, 422, msg);
+  }
+
+  // Check that campaign suports requested message type.
+  const messageBody = campaign.messages[type];
+  if (!messageBody) {
+    const msg = `Campaign ${campaignId} does not support reminders for '${type}'`;
+    return helpers.sendResponse(res, 422, msg);
+  }
+
+  return app.locals.clients.northstar.Users.get('mobile', phone)
+  .then((user) => {
+    mobilecommons.send_message(phone, messageBody);
+    const msg = `Sent text for ${campaignId} ${type} to ${user.mobile}`;
```

Comment:
We should avoid logging mobile numbers when possible. I'll be changing this to output the user.id in a future PR